### DISCUSSION TITLE
Consumer Partition Auto Assignment

### DISCUSSION THREAD

**Discussion by Navin009** [fluvio-community/fluvio#4178] [Q&A]
Unable to find the consumer auto partition assignment.
Anybody can help me know how to setup fluvio consmer with service that is scalable, 
Currenly i have 3 partition, service have 2 pods running, pods will scale up to 5 pods.

**Comment by sehz** [comment]
Hi,  by default, fluvio produce and consume will use [round robin](https://www.fluvio.io/docs/fluvio/cli/fluvio/produce#producing-to-multiple-partitions-using-round-robin) strategy.   That should be sufficient for most of use cases.   

So that means, for your case
- Create topic with 3 partitions [See CLI](https://www.fluvio.io/docs/fluvio/cli/fluvio/topic).   Make sure you have at least have 2 SPU.
- Service will use default consumer so it will use available partitions.

Only limitation is that consumer will not detect new partition.   But work around is to update service deployment when partition is added.
